### PR TITLE
heimdall-7 

### PR DESCRIPTION
**Pull request by aakash17kc**
The groupId for Heimdall was changed from com.miq.automation to com.miqdigital.automation.

Work for #7

### CODE REVIEW

File: heimdall.application/pom.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
         <artifactId>heimdall</artifactId>
-        <groupId>com.miq.automation</groupId>
+        <groupId>com.miqdigital.automation</groupId>
         <version>1.0-SNAPSHOT</version>
```
Comment (ntantri) [inline heimdall.application/pom.xml]: Can we change the snapshot to a release version of 1.0?

File: heimdall.sample-client/pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
 
     <parent>
         <artifactId>heimdall</artifactId>
-        <groupId>com.miq.automation</groupId>
+        <groupId>com.miqdigital.automation</groupId>
         <version>1.0-SNAPSHOT</version>
```
Comment (ntantri) [inline heimdall.sample-client/pom.xml]: Can we keep this as well for version as 1.0 ?

File: pom.xml [hunk 3]
```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.miq.automation</groupId>
+    <groupId>com.miqdigital.automation</groupId>
     <artifactId>heimdall</artifactId>
     <packaging>pom</packaging>
     <version>1.0-SNAPSHOT</version>
```
Comment (ntantri) [inline pom.xml]: Similar

**Review by ntantri — CHANGES_REQUESTED** [review]

**Comment by ntantri** [comment]
@aakash17kc can you please create a ticket for this and add it under Issues?

File: heimdall.application/pom.xml (hunk 1 above)
Comment (aakash17kc) [reply heimdall.application/pom.xml]: I will update it.

**Comment by aakash17kc** [comment]
> @aakash17kc can you please create a ticket for this and add it under Issues?

It is the part of "Add Heimdall on Maven Central" issue. Attaching the link - https://github.com/MediaIQ/heimdall/issues/7

**Review by vaibsaraswat — APPROVED** [review]

**Review by ntantri — APPROVED** [review]

**Pull request merged**